QA Notice: Compatibility with CMake < 3.5 has been removed from CMake 4

```
 * QA Notice: Compatibility with CMake < 3.5 has been removed from CMake 4,
 * dev-libs/nanopb will fail to build w/o a fix.
 * See also tracker bug #951350; check existing or file a new bug for this package.
 * Please also take it upstream.
 * 
 * The following files are causing errors:
 *   examples/cmake_simple/CMakeLists.txt:2.8
 *   examples/cmake_relpath/CMakeLists.txt:2.8
 *   conan-wrapper/CMakeLists.txt:2.8.12
 *   build-tests/legacy_cmake_simple/CMakeLists.txt:2.8
 *   build-tests/legacy_cmake_relpath/CMakeLists.txt:2.8
 *   build-tests/cmake_with_components/CMakeLists.txt:2.8
 * 
 * Compatibility with CMake < 3.10 will be removed in a future release.
 * See also tracker bug #964405; check existing or file a new bug for this package.
 * If not fixed in upstream's code repository, we should make sure they are aware.
 * 
 * CMake 4 detected; building with -DCMAKE_POLICY_VERSION_MINIMUM=3.5
 * This is merely a workaround to avoid CMake Error and *not* a permanent fix;
 * there may be new build or runtime bugs as a result.
 * 
 * An upstreamable patch should take any resulting CMake policy changes
 * into account. See also:
 *   https://cmake.org/cmake/help/latest/manual/cmake-policies.7.html

```

--- a/build-tests/cmake_with_components/CMakeLists.txt
+++ b/build-tests/cmake_with_components/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(NANOPB_CMAKE_SIMPLE C CXX)
 
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../extra)

--- a/build-tests/legacy_cmake_relpath/CMakeLists.txt
+++ b/build-tests/legacy_cmake_relpath/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(NANOPB_CMAKE_SIMPLE C)
 
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../extra)

--- a/build-tests/legacy_cmake_simple/CMakeLists.txt
+++ b/build-tests/legacy_cmake_simple/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(NANOPB_CMAKE_SIMPLE C)
 
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../extra)

--- a/conan-wrapper/CMakeLists.txt
+++ b/conan-wrapper/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.5)
 project(cmake_wrapper)
 
 include(${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake)

--- a/examples/cmake_relpath/CMakeLists.txt
+++ b/examples/cmake_relpath/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(NANOPB_CMAKE_SIMPLE C)
 
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../extra)

--- a/examples/cmake_simple/CMakeLists.txt
+++ b/examples/cmake_simple/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(NANOPB_CMAKE_SIMPLE C)
 
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../extra)

